ramalama/cli.py

```diff
@@ -189,6 +189,15 @@ def configure_arguments(parser):
         action="store_true",
         help="offload the workload to the GPU",
     )
+    parser.add_argument("--ngl", dest="ngl", type=int, default=999, help="Number of layers to offload to the gpu, if available")
+    parser.add_argument(
+        "--keep-groups",
+        dest="podman_keep_groups",
+        default=False,
```

Comment:
Add function to detect, also option should be configurable in ramalama.conf file.

Reply:
What is the right way to detect a rootless podman? I found how to detect podman by `os.path.basename(args.engine) == "podman"`, but I'm not sure about rootless